llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -1080,6 +1080,21 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
       }
     }
 
+    // TODO: Could we merge some code with zvfhmin?
+    if (Subtarget.hasVInstructionsBF16()) {
+      for (MVT VT : BF16VecVTs) {
+        if (!isTypeLegal(VT))
+          continue;
+        setOperationAction({ISD::FP_ROUND, ISD::FP_EXTEND}, VT, Custom);
+        setOperationAction({ISD::VP_FP_ROUND, ISD::VP_FP_EXTEND}, VT, Custom);
+        setOperationAction({ISD::CONCAT_VECTORS, ISD::INSERT_SUBVECTOR,
+                            ISD::EXTRACT_SUBVECTOR, ISD::SCALAR_TO_VECTOR},
+                           VT, Custom);
+        setOperationAction({ISD::LOAD, ISD::STORE}, VT, Custom);
+        // TODO: Promote to fp32.
```

Comment:
It looks like F16Minimal has support for `STRICT_FP_ROUND,  STRICT,FP_EXTEND, VP_MERGE, VP_SELECT, SELECT, SELECT_CC, SINT_TO_FP, UINT_TO_FP, VP_SINT_TO_FP, VP_UINT_TO_FP, and SPLAT_VECTOR` but these are not included for BF16. Do you plan to add support for these?

Reply:
Zvfbfmin does not have any integer conversion instructions so `SINT_TO_FP, UINT_TO_FP, VP_SINT_TO_FP, VP_UINT_TO_FP` can be removed.

Reply:
I'm curious what SPLAT_VECTOR for F16Minimal does since the scalar FP16 type isn't guaranteed legal with F16Minimal.

Reply:
> I'm curious what SPLAT_VECTOR for F16Minimal does since the scalar FP16 type isn't guaranteed legal with F16Minimal.

I create a pr https://github.com/llvm/llvm-project/pull/88275 to add the constraint for `SPLAT_VECTOR`.

Reply:
Based on the comments in this thread, we have been able to trim down the original list. We are left with consideration of `STRICT_FP_ROUND,  STRICT_FP_EXTEND. Are we able to address those opcodes here?

Reply:
> Based on the comments in this thread, we have been able to trim down the original list. We are left with consideration of `STRICT_FP_ROUND, STRICT_FP_EXTEND`. Are we able to address those opcodes here?

Added `STRICT_FP_ROUND`, ` STRICT_FP_EXTEND` now.